include/deal.II/base/vectorization.h

```diff
@@ -224,6 +224,20 @@ class VectorizedArray
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
 
+  VectorizedArray()
+  {}
+
+  VectorizedArray(const Number scalar)
+  {
+    this->operator=(scalar);
+  }
+
+  void
+  broadcast_from_scalar(const Number scalar)
```

Comment:
@kronbichler @tjhei Do we want to deprecate `operator=(const Number)` then? I don't see a reason for having to functions doing exactly the same (both conceptually and behaviorwise).

Reply:
We need to provide `operator=` for compatibility in the various tensor classes where we use it to set to zero. So that part is fundamental to the class and the compatibility we want to achieve. The question is whether we should omit this `broadcast_from_scalar` then?